skimage/morphology/convex_hull.py

```diff
@@ -115,7 +116,13 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
             # when offsetting, we multiply number of vertices by 2 * ndim.
             # therefore, we reduce the number of coordinates by using a
             # convex hull on the original set, before offsetting.
-            hull0 = ConvexHull(coords)
+            try:
+                hull0 = ConvexHull(coords)
+            except QhullError as err:
+                warn("Failed to get convext hull image"
+                     "Returning empty image, see error messege below"
+                     "%s" % str(err), UserWarning)
```

Comment:
```suggestion
                warn(f"Failed to get convex hull image"
                     f"Returning empty image, see error message below\n"
                     f"{err}")
```
fix typos and use f-strings